fe/fe-core/src/main/java/com/starrocks/qe/ConnectContext.java

```diff
@@ -672,11 +672,11 @@ public void setResourceGroup(TWorkGroup resourceGroup) {
     }
 
     public String getCurrentCatalog() {
-        return currentCatalog;
+        return this.sessionVariable.getCatalog();
     }
 
     public void setCurrentCatalog(String currentCatalog) {
-        this.currentCatalog = currentCatalog;
+        this.sessionVariable.setCatalog(currentCatalog);
```

Comment:
If this function returns a session variable, the currentCatalog will be useless. It will be error-prone to have two variables that represent the same thing.

Reply:
`ConnectContext.currentCatalog` removed